src/app/shared/components/addscenario/addscenario.component.ts

```diff
@@ -42,8 +43,9 @@ export class AddScenarioModal implements OnInit, OnDestroy {
     public cloneParameters: any;
     public regRegion: any;
     public statisticGroup: any;
-    public clone: boolean;
+    public clone: boolean = false;
     public selectedRegion;
+    public orgRegion;
```

Comment:
Is this short for "original region"? maybe this should be changed to `originalRegion`